.travis.yml

```diff
@@ -2,15 +2,14 @@ sudo: false
 language:
   - python
 python:
-  - "3.5"
+  - 3.5
```

Comment:
This was correct originally. It should be `"3.5"`, not `3.5`, otherwise YAML will treat it as a floating point number, which might work with Travis, but only because the folks at Travis are nice.